.github/automation/build_linters.sh

```diff
@@ -0,0 +1,41 @@
+# Build oneDNN for PR linter checks.
```

Comment:
Thanks for adding this one. I do not want to stall this PR with discussion on build script design, so feel free to address this feedback separately:
* I would suggest to have a single build script for CI with command line or envvar interface based on [automation/build.sh|https://github.com/oneapi-src/oneDNN/blob/main/.github/automation/build.sh].
* Please avoid dependency on Github job name in the script as these can change resulting in nasty issues. Ideally the build script should be straightforward to use outside of Github actions.
*